Pipfile

```diff
@@ -5,13 +5,18 @@ name = "pypi"
 
 [packages]
 python-dotenv = "~=1.0"
-requests = "~=2.26"
 ruff = "~=0.7"
 tqdm = "~=4.66"
+requests = "*"
+wget = "*"
```

Comment:
Also, please specify a loose version requirement, like the other libraries in this file.

`wget = "~=9.9"`

Or whatever the current major/minor version is. You can get it off pypi, or with `pipenv run pip freeze`. Don't specify the patch version (so 9.9, not 9.9.9), so that we are pegged to the major version and the minor version can increment (versus being pegged to the minor version and only the patch version can increment). For more information see https://semver.org/